usr/sbin/pkcsslotd/slotmgr.c

```diff
@@ -467,6 +468,13 @@ static int slotmgr_key_vers(void *private, int tok, unsigned int vers)
     return 1;
 }
 
+static void slotmgr_disab_event_supp(void *private)
+{
+    UNUSED(private);
```

Comment:
In the other parser callbacks, `private` is used instead of global variables inside `slotmgr` (as far as I can see and remember).  Shouldn't we keep this pattern here, too?  It was initially intended for some unit tests, but I did not yet find the time to write them...

Reply:
Right, but for example, in `slotmgr_end_slot()` it first completes the slot inside the private data, but then moves the finalized slot entry into the next slot entry in the global `sinfo` array. So the result also ends up in a global variable. 

I could certainly add a pointer to the global variable event_support_disabled in the private data, and update it that way, but does this really make a difference?

Reply:
Since I do not know when/if I get around to writing these tests, lets not change anything here.